client/my-sites/google-my-business/select-business-type/index.jsx

```diff
@@ -0,0 +1,657 @@
+											stroke-linecap="round"
+											stroke-linejoin="round"
+										/>
+									</g>
+									<path
+										d="M56.3782812,22.0091832 C55.9079687,21.2466832 55.3345312,20.6044957 54.7071875,20.0982457 C54.1392187,19.6388707 53.75875,18.999027 53.649375,18.277152 C53.4978125,17.2779332 53.1485937,16.3099645 52.6735937,15.6193395 C52.015,14.661527 50.8821875,13.9232457 49.7001562,13.6193395 C48.6267187,11.714652 47.6579687,11.4349645 46.30875,11.030277 C44.0884375,10.3638707 42.0157812,11.6224645 42.0157812,11.6224645 C41.7204687,14.0849645 42.6892187,16.852152 44.99625,17.0076207 C44.5954687,18.8849645 45.6798437,21.4568395 47.7728125,22.0701207 C47.549375,23.4857457 47.8829687,25.1560582 48.82125,26.6748082 C50.5415625,29.4623082 53.6282812,30.677152 55.7157812,29.389652 C57.8025,28.1005895 58.099375,24.7966832 56.3782812,22.0091832"
+										id="Fill-369"
+										fill="#005081"
+									/>
+									<path
+										d="M33.6491406,27.9594176 L31.2069531,21.0125426 C31.0694531,20.6219176 31.2741406,20.1945739 31.6647656,20.0570739 L35.0163281,18.8781676 C35.4069531,18.7414489 35.8342969,18.9469176 35.9717969,19.3367614 L38.4147656,26.2836364 C38.5514844,26.6742614 38.3467969,27.1023864 37.9561719,27.2398864 L34.6046094,28.4180114 C34.2147656,28.5555114 33.7866406,28.3500426 33.6491406,27.9594176"
+										id="Fill-452"
+										fill="#00A9DE"
+									/>
+									<polygon
+										id="Fill-453"
+										fill="#FFFFFF"
+										points="37.5626562 25.8181676 34.0220312 27.0626989 31.8954687 21.0150426 35.4360937 19.7705114"
+									/>
+									<polygon
+										id="Fill-454"
+										fill="#004F84"
+										points="35.4290625 19.750902 31.8884375 20.9962145 32.1845312 21.8368395 35.724375 20.5923082"
+									/>
+									<g id="Group-457" transform="translate(34.312500, 26.486293)">
+										<path
+											d="M1.62320313,1.54515625 C1.62320313,1.54515625 0.972421875,-0.0275 0.716953125,0.21078125 C0.164609375,0.7271875 0.831796875,2.88265625 0.831796875,2.88265625"
+											id="Fill-455"
+											fill="#C7D6E1"
+										/>
+										<path
+											d="M1.62320313,1.54515625 C1.62320313,1.54515625 0.972421875,-0.0275 0.716953125,0.21078125 C0.164609375,0.7271875 0.831796875,2.88265625 0.831796875,2.88265625"
+											id="Stroke-456"
+											stroke="#005081"
+											stroke-linecap="round"
+											stroke-linejoin="round"
+										/>
+									</g>
+								</g>
+							</g>
+						</g>
+					</svg>
+				</Card>
+
+				<CTACard
+					headerText={ translate( 'Physical Location or Service Area' ) }
+					mainText={ translate(
+						'My business has a physical location custmers can visit, ' +
+							'or provides goods and services to local customers, or both.'
+					) }
+					buttonText={ translate( 'Create My Listing' ) }
+					buttonIcon="external"
+					buttonPrimary={ true }
+				/>
+
+				<CTACard
+					headerText={ translate( 'Online Only' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Online' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gmb-select-biz-type)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).